.vscode/launch.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": ".NET Core Launch (console)",
+            "type": "coreclr",
+            "request": "launch",
+            "preLaunchTask": "build",
+            "program": "${workspaceFolder}/src/Analyzer.Cli/bin/Debug/net5.0/TemplateAnalyzer.dll",
+
+            // Runs the CLI against the currently open ARM template file.
```

Comment:
You mean the currently open template in VSCode?

Reply:
Yes - I'll update to be more clear.